src/main/java/com/internship/backend/model/DateRangeRequest.java

```diff
@@ -1,18 +1,5 @@
 package com.internship.backend.model;
 
-import lombok.AllArgsConstructor;
-import lombok.Getter;
-import lombok.NoArgsConstructor;
-import lombok.Setter;
-//
-//@Setter
-//@Getter
-//@AllArgsConstructor
-//@NoArgsConstructor
-//public class DateRangeRequest {
-//
-//    private NewDate startDate;
-//    private NewDate endDate;
-//}
+import java.time.LocalDateTime;
 
-public record DateRangeRequest(NewDate start, NewDate end) {}
+public record DateRangeRequest(LocalDateTime start, LocalDateTime end) {}
```

Comment:
what is the purpose of this DateRangeRequest in the model package?

Reply:
I use DateRangeRequest in ReservationController in getAvailableTennisCourts(@RequestBody DateRangeRequest dateRangeRequest). It contains both startTime and endTime.
Should I use directly LocalDateTime instead of DateRangeRequest?

Reply:
Yes, I think it would be a better option